src/main/scala/no/ndla/mapping/License.scala

```diff
@@ -49,7 +50,8 @@ object License extends Enumeration {
     LicenseDefinition(
       CC_BY_NC_ND,
       "Creative Commons Attribution-NonCommercial-NoDerivs 4.0 International",
-      Some("https://creativecommons.org/licenses/by-nc-nd/4.0/"))
+      Some("https://creativecommons.org/licenses/by-nc-nd/4.0/")),
+    LicenseDefinition(NA, "Ikke relevant", None),
```

Comment:
Bør gjerne ha en internasjonal standardverdi.
```suggestion
    LicenseDefinition(NA, "Not Appliccable", None),
```